src/components/add/Form.tsx

```diff
@@ -36,8 +37,9 @@ export default function Form({ setError, selectedWallet, userAccount }: Attrs) {
   const [isFocused, setIsFocused] = useState(false);
   const [enableEncryption, setEnableEncryption] = useState(false);
   const [code, setCode] = useState("");
+  const confirm = useConfirm();
 
-  async function addNote(data: FormData) {
+  const addNote = async (data: FormData) => {
```

Comment:
What is the purpose of this change?

Reply:
To capture `confirm` from the parent scope